source/educators/how-tos/open_response.rst

```diff
@@ -1,3 +1,516 @@
 Add an Open Response Assessment
-#################################
+###############################
```

Comment:
This is a really long document. I think it would benefit from at minimum a table of contents. I wonder if this is multiple How-To documents covering different use cases, it seems like there's lots of branching logic.

I would wait for @feanil to get a chance to review before beginning to change things though.

Reply:
Yeah, I would honestly like to restructure and rewrite this whole thing. This was an attempt to just get *something* in our docs instead of an empty doc but probably better to build up from nothing.